javascript/javascript2/week2/hyfindex.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="hyfmain.css">
+    <title>hyfBayHelpers-Js2Week2</title>
+</head>
+<body>
+  <header>
+    <div class="nav">
+      <img class="logo" src="https://thumbs.dreamstime.com/z/new-product-item-store-25048588.jpg" alt="Yellow new product bage">
+      <a href="#">Home</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+    </div>
+  </header>
+  <main> 
+    <div id="search-wrapper"> 
+      <input type="search" id="search-bar"  placeholder="Search for a product">
+      <input type="number" id="max-price"  placeholder="Set Maximum price">
```

Comment:
code looks good.
however : These two number fields allowing user to enter -values. you might restrict user entering minus numbers.